build.rs

```diff
@@ -5,4 +5,6 @@ fn main() {
     println!("cargo:rustc-link-lib=dylib=cudart");
     println!("cargo:rustc-link-lib=dylib=cublas");
     println!("cargo:rustc-link-lib=dylib=cublasLt");
+    println!("cargo:rustc-link-search=native={}", std::env::var("CUDNN_LIB").expect("Failed to read environmental variable `CUDNN_LIB`. Set `CUDNN_LIB` to `path/to/cudnn/lib`."));
```

Comment:
Do you know why this is needed for cudnn but not other libs?

Reply:
because cudnn is installed separately and is not in the folder where cuda resides

Reply:
Ooooh did not realize that

Reply:
So we'll probably need to wrap this functionality with a feature if it's not installed by default with cuda. Will investigate a bit on my dev machine to see if it's installed/what the process is like. For dfdx I wonder if we should require cudnn, or fallback to custom kernels if people don't have cudnn installed

Reply:
does something like pytorch or tensorflow work with cuda but without cudnn? i don't think so and i don't see any reason why someone could install cuda but not cudnn ...

Reply:
I think pytorch bundles cuda/cudnn libraries together? Not completely sure tbh. If cudnn is installed by default then i don't think we need feature flag. We can just require it but I want to go through install process to see how annoying it is first

Reply:
not sure about pytorch, but it is seperate in tensorflow (you have to install everything yourself anyway)...